migrations/sqls/20211017120050-add-badge-caching-up.sql

```diff
@@ -0,0 +1,21 @@
+create table badge_group (
+  id uuid primary key,
+  group_name text
```

Comment:
No `slug`? Also maybe the name shouldn't be `slug`, because it's more like the name of the function to call to compute progress on a badge. `function_name`?

Reply:
My bad, I think I looked at a very outdated version of the issue and didn't hit refresh when I did this. Only now I found the recent changes. Thank you for catching that and sorry as well.